libkbfs/folder_block_ops.go

```diff
@@ -952,10 +1011,6 @@ func (fbo *folderBlockOps) fixChildBlocksAfterRecoverableErrorLocked(
 			df.setBlockOrphaned(oldPtr, false)
 		}
 	}
-	if df == nil || !df.isBlockDirty(file.tailPointer()) ||
```

Comment:
This check turned out to be unnecessary, and actually harmful now that there can be multiple levels of indirection.  If this recovery function is called, we already know for sure that the root file block was dirty and syncing; however a retried sync following a deferred write (as in `TestKBFSOpsMultiBlockWriteDuringRetriedSync`) could incorrectly return when the root block has already been copied (and thus isn't "dirty" by the definition), and prevent other indirect blocks from being recovered below.